plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -246,7 +245,7 @@ const uploadPrivateKey = async (sshkeyManager: SshKeyManager) => {
         hostName = `default-${Date.now()}`;
     }
 
-    const tempDir = mkdtempSync(join(os.tmpdir(), 'private-key-'));
+    const tempDir = await mkdtemp(join(os.tmpdir(), 'private-key-'));
```

Comment:
Not a fan of removing the "sync" here. I don't know how to test this change and making something async _can_ lead to concurrency problems.

Reply:
what is the difference between
`await somethingASync` vs `somethingSync` ?

Reply:
also I don't see what concurrency problems would be there